entity-management-tests/src/integration-test/java/eu/europeana/entitymanagement/testutils/IntegrationTestUtils.java

```diff
@@ -213,8 +213,8 @@
           ORGANIZATION_NATURALIS_ZOHO_RESPONSE,
           ORGANIZATION_GFM_URI_ZOHO,
           ORGANIZATION_GFM_ZOHO_RESPONSE,
-          ORGANIZATION_PCCE_URI_ZOHO,
-          ORGANIZATION_PCCE_ZOHO_RESPONSE,
+          ORGANIZATION_BYZART_URI_ZOHO,
```

Comment:
please add back the PCCE organization as it is used in integration tests

Reply:
Added back.